test/tests/expected/global_deps_slim.js

```diff
@@ -0,0 +1,8 @@
+[
+    global,
+    function (stealRequire, stealExports) {
+        stealRequire(1);
+        stealRequire(2);
+        var GLOBAL = 'I don\'t like "Quotes"';
```

Comment:
These types of globals won't work with slim (and I don't expect them to), only the `window.GLOBAL` variety.   Will your transform handle those as well?